Mod.cs

```diff
@@ -1,48 +1,58 @@
-using KitchenLib;
-using KitchenLib.Logging;
 using KitchenMods;
-using System.Reflection;
 using System.Collections.Generic;
-using System;
 using System.Linq;
 using KitchenDesignManager.Providers;
+using PreferenceSystem;
+using ONe.KitchenDesigner.KitchenDesigns;
+using Kitchen;
+using ExitGames.Client.Photon.StructWrapping;
 
 
 namespace KitchenDesignManager
 {
-    public class Mod : BaseMod, IModSystem
+
+    public class Mod : IModInitializer
     {
         public const string MOD_GUID = "com.ohkannaduh.kdm";
         public const string MOD_NAME = "Kitchen Design Manager";
-        public const string MOD_VERSION = "1.0.0";
-        public const string MOD_AUTHOR = "OhKannaDuh";
-        public const string MOD_GAMEVERSION = ">=1.1.4";
-
-        public static List<Entry> Entries = new List<Entry>();
-
-        public static KitchenLogger Logger;
-
-        public Mod() : base(MOD_GUID, MOD_NAME, MOD_AUTHOR, MOD_VERSION, MOD_GAMEVERSION, Assembly.GetExecutingAssembly())
-        {
-        }
-
-        protected override void OnInitialise()
+        public const string MOD_VERSION = "1.1.0";
```

Comment:
Due to the JSON layout changing this should be 2.0.0 since it's a breaking change. This gives me an opportunity to make another change that I had held back on.

Reply:
Yeah, didn't think of that. You're right, definitly 2.0.0.